integration-tests/src/test/java/org/zowe/apiml/integration/authentication/services/ZaasClientIntegrationTest.java

```diff
@@ -143,7 +143,7 @@ static Stream<Arguments> provideInvalidAuthHeaders() {
         return Stream.of(
             Arguments.of(getAuthHeader(INVALID_USER, PASSWORD), ZaasClientErrorCodes.INVALID_AUTHENTICATION),
             Arguments.of(getAuthHeader(NULL_USER, PASSWORD), ZaasClientErrorCodes.INVALID_AUTHENTICATION),
-            Arguments.of(getAuthHeader(EMPTY_USER, PASSWORD), ZaasClientErrorCodes.INVALID_AUTHENTICATION),
+            Arguments.of(getAuthHeader(EMPTY_USER, PASSWORD), ZaasClientErrorCodes.EMPTY_NULL_USERNAME_PASSWORD),
```

Comment:
Is it intentional? It looks here as more detail in the error (which is wrong), but maybe against the main branch?

Reply:
If I remember correctly I found some inconsistency with the returned message also on the main branch. that was the reason of changing it